### PR TITLE
[9.0][MIG] account_banking_pain_base: Script OpenUPgrade

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @tecnativa

### CODE REVIEW

File: account_banking_pain_base/migrations/9.0.1.0.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    # Copy convert_to_ascii to account_payment_method
+    sql = """
+    UPDATE account_payment_method apm
+    SET convert_to_ascii=am.convert_to_ascii
+    WHERE apm.payment_method_id = am.id
```
Comment (pedrobaeza) [inline account_banking_pain_base/migrations/9.0.1.0.0/post-migration.py]: You have missed `FROM` line

**Comment by sergio-teruel** [comment]
@pedrobaeza changes done

**Review by pedrobaeza — APPROVED** [review]
With latest changes

**Comment by pedrobaeza** [comment]
Merging optimistically as OpenUpgrade policy (although it has been already tested in a migration)

**Pull request merged**